### PR TITLE
Add PPA installation instructions to the documentation

### PR DESCRIPTION
**Pull request by clinssen**
@steffengraber: do you know for which Linux distributions this is known to work? Ubuntu is confirmed, I guess it should work on Debian and Mint too?

### CODE REVIEW

**Comment by steffengraber** [comment]
> @steffengraber: do you know for which Linux distributions this is known to work? Ubuntu is confirmed, I guess it should work on Debian and Mint too?

First of all, only Ubuntu. 
I have to test everything else first. But the step to support several Linux systems would be nice.

File: doc/installation.rst [hunk 1]
```diff
@@ -16,7 +16,30 @@ NESTML can then be installed into your local user directory via:
...
+
+   sudo add-apt-repository ppa:nest-simulator/nest
+
+Then update the index and install the necessary packages:
+
+.. code-block:: bash
+
+   sudo apt update
+   sudo apt install nest python3-nestml
+
```
Comment (steffengraber) [inline doc/installation.rst:37]: A few python dependencies still need to be installed:

`python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10`

**Review by steffengraber — APPROVED** [review]
:+1:  Approved a small addition.

**Comment by steffengraber** [comment]
> Similarly, how feasible is it to create a `brew` package for macOS?
I'm not an expert in macOS, but it's doable. Do you want to create one, or should I take a look?

File: doc/installation.rst (hunk 1 above)
Comment (clinssen) [reply doc/installation.rst:37]:
```suggestion
python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10

```

File: doc/installation.rst [hunk 2]
```diff
@@ -16,7 +16,31 @@ NESTML can then be installed into your local user directory via:
...
+   sudo add-apt-repository ppa:nest-simulator/nest
+
+Then update the index and install the necessary packages:
+
+.. code-block:: bash
+
+   sudo apt update
+   sudo apt install nest python3-nestml
+python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10
```
Comment (clinssen) [inline doc/installation.rst:37]:
```suggestion
   python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10
```

**Pull request merged**